packages/ingest/src/granule.ts

```diff
@@ -1,71 +1,109 @@
-'use strict';
+import * as errors from '@cumulus/errors';
+import moment from 'moment';
+import * as S3 from '@cumulus/aws-client/S3';
+import * as log from '@cumulus/common/log';
+import { DuplicateHandling } from '@cumulus/types';
+
+export interface EventWithDuplicateHandling {
+  config: {
+    collection: {
+      duplicateHandling?: DuplicateHandling
+    },
+    duplicateHandling?: DuplicateHandling,
+  },
+  cumulus_config?: {
+    cumulus_context?: {
+      forceDuplicateOverwrite?: boolean
+    }
+  },
+}
 
-const encodeurl = require('encodeurl');
-const errors = require('@cumulus/errors');
-const get = require('lodash/get');
-const moment = require('moment');
-const S3 = require('@cumulus/aws-client/S3');
-const log = require('@cumulus/common/log');
-const { deprecate } = require('@cumulus/common/util');
+export interface File {
```

Comment:
Given the likelihood of using this multiple places (the API package code?), should it go into the types package?

Reply:
Probably, but I want to finish migrating the rest of the ingest package first to really flesh out what this type really is. I suspect it may end up getting broken into other types before this is all said and done. Leaving it here for now makes it easier to change.